transit/world_feed/world_feed_tests/subway_converter_tests.cpp

```diff
@@ -0,0 +1,385 @@
+#include "testing/testing.hpp"
+
+#include "transit/world_feed/subway_converter.hpp"
+#include "transit/world_feed/world_feed.hpp"
+
+#include "platform/platform.hpp"
+
+#include "base/assert.hpp"
+#include "base/file_name_utils.hpp"
+
+#include <fstream>
+#include <string>
+#include <utility>
+#include <vector>
+
+namespace
+{
+std::string const kSubwayTestsDir = "transit_subway_converter_tests";
+std::string const kSubwayJsonFile = "subways.json";
+std::string const kMappingFile = "mapping.txt";
+
+void WriteStringToFile(std::string const & fileName, std::string const & data)
+{
+  std::ofstream file;
+  file.open(fileName);
+  file << data;
+}
+}  // namespace
+
+namespace transit
+{
+class SubwayConverterTests
+{
+public:
+  SubwayConverterTests()
+    : m_mwmMatcher(GetTestingOptions().m_resourcePath, false /* haveBordersForWholeWorld */)
+  {
+    auto const & options = GetTestingOptions();
+    GetPlatform().SetResourceDir(options.m_resourcePath);
+
+    CHECK(Platform::MkDirChecked(kSubwayTestsDir), ("Could not create directory for test data"));
```

Comment:
`CHECK(Platform::MkDirChecked(kSubwayTestsDir), ("Could not create directory for test data:", kSubwayTestsDir));`?